### PR TITLE
Preparation for multiple icon styles / improved code

### PR DESCRIPTION
**Pull request by annando**
This PR contains the preparation for multiple styles, so you don't only have got the coice between "color" and "black".

Also the whole code is rewritten and improved.

### CODE REVIEW

File: images/platforms/color/sharkey.license
Comment (MrPetovan) [inline images/platforms/color/sharkey.license:1]: This file should be renamed to `sharkey.svg.license` to follow the convention used in this folder.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: images/platforms/color/sharkey.license
Comment (annando) [reply images/platforms/color/sharkey.license:1]: The file is removed.

**Comment by annando** [comment]
I now added the icons from https://codeberg.org/FediverseIconography/pages This should now be a mostly complete list.

**Comment by annando** [comment]
Thanks to @loma-one mow white icons are added as well.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**